tcl/helpbrowser.tcl

```diff
@@ -30,17 +30,21 @@ proc ::helpbrowser::open_helpbrowser {} {
             .helpbrowser configure -menu $::dialog_menubar
         }
 
-        # set the maximum number of child columns to create
-        set ::helpbrowser::maxcols 5
+        # set the maximum number of parent columns to create
+        set ::helpbrowser::maxcols 4
 
-        # TODO wrap frame in a canvas with a horz scrollbar,
-        # currently we simply add cols to the left until we reach max cols
+        # only make x-axis scrollable
         wm resizable .helpbrowser 0 1
         ::helpbrowser::make_frame .helpbrowser
 
         # hit up, down, or Tab after browser opens to focus on first listbox
-        bind .helpbrowser <KeyRelease-Up> "focus .helpbrowser.frame.root0"
-        bind .helpbrowser <KeyRelease-Down> "focus .helpbrowser.frame.root0"
+        bind .helpbrowser <KeyRelease-Up> "focus .helpbrowser.c.f.root0"
+        bind .helpbrowser <KeyRelease-Down> "focus .helpbrowser.c.f.root0"
+
+        # ignore undo bindings?
```

Comment:
This seems to be an extra copy/paste

Reply:
I just kept what was already there. what do you mean?

Reply:
I see this section twice in my local checkout and the separate file view, just not in the diff:

<img width="612" alt="Screen Shot 2022-08-15 at 1 09 26 AM" src="https://user-images.githubusercontent.com/480637/184558294-4bdfdab7-394c-475a-9134-f6bd44418289.png">

Reply:
ohh must've happened bc the changes from #1481 got merged twice

Reply:
@danomatika think I got it